.github/workflows/dispatch_deploy.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Deploy environment'
+        required: true
+        default: 'dev'
+        type: choice
+        options:
+        - dev
+        - stage
+        - prod
+      cognito_app_secret:
+        description: 'AWS Cognito secret store'
+        type: string
+        required: false
+        default: veda-auth-stack-alukach/veda-workflows
+      stac_ingestor_url:
+        description: 'URL for the stac ingestor API. (A deployment of https://github.com/NASA-IMPACT/veda-stac-ingestor)'
+        required: true
+        type: string
+      external_role_arn:
+        description: 'ARN refering to the external IAM role which is assumed during ingests.'
+        required: true
+        type: string
+
+jobs:
+  deploy:
+    uses: ./.github/workflows/deploy.yml
+    with:
+      deploy_env: ${{ inputs.environment }}
```

Comment:
```suggestion
      environment: ${{ inputs.environment }}
```

Reply:
Thought I'd cleared this up. Good catch.